packages/flutter/lib/src/widgets/autofill.dart

```diff
@@ -109,13 +140,19 @@ export 'package:flutter/services.dart' show AutofillHints;
 ///  }
 /// ```
 /// {@end-tool}
+///
+/// See also:
+///
+/// * [AutofillContextAction], an enum that contains predefined autofill context
+///   clean up actions to be run when a topmost [AutofillGroup] is disposed.
 class AutofillGroup extends StatefulWidget {
   /// Creates a scope for autofillable input fields.
   ///
   /// The [child] argument must not be null.
   const AutofillGroup({
     Key key,
     @required this.child,
+    this.onDisposeAction = AutofillContextAction.commit,
```

Comment:
assert that this is not null?

Reply:
It's nullable, and on dispose action is going to be a no-op when it's null.

Reply:
We should document what no-op in that case means (and instead of using null, can we define a `AutofillContextAction ` for that?

Reply:
I wonder why is a separate enum case preferred over `null`? I think in this case using null to stand for "no action should be taken" makes sense too. Is it because the analyzer has exhaustiveness check for enum cases?